apic_ml2/neutron/plugins/ml2/drivers/cisco/apic/mechanism_apic.py

```diff
@@ -858,10 +858,10 @@ def _perform_path_port_operations(self, context, port):
                 app_profile_name=self._get_network_app_profile(
                     context.network.current), transaction=trs)
 
-    def _perform_interface_port_operations(self, context, port,
+    def _perform_interface_port_operations(self, context, port, network,
                                            is_delete=False):
         router_id = port.get('device_id')
-        if router_id == '':
+        if not router_id:
```

Comment:
Is it important here to distinguish between None and empty-string (i.e. is there a reason why it was empty string)?